### PR TITLE
Added 4 more Impact-Component cards

### LINKED ISSUE #47: Add more cards to impact section
This is the impact section on homepage currently having 2 cards, 
image

You need to add 4 more cards to it, of similar dimensions, 
1. Open-Source Journey (Get your journey in open-source started with our open-source projects)
2. Daily Leetcoding (Solve Leetcode challenges every day with our coders) 
3. Get Career guidance (Confused about your interest/career in tech, Get Personal guidance from experts) 
4. Build good Linkedin Profile (Learn how to make an impressive profile) 

The issue is tagged medium because you yourself need to figure out the illustrations(images) for each card.

### PR DESCRIPTION
**Pull request by Meghag5**
(feature/Impact-Component)-Added Impact component

# Description

Added 4 more cards Impact Component 

Fixed #47 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Components/Impact.js [hunk 1]
```diff
@@ -33,6 +33,50 @@ export default class Impact extends Component {
                 para2: "right way!",
               }}
             />
+          </Row>
+          <Row
+            className="justify-content-md-center"
+            style={{ marginTop: "5px" }}
+          >
```
Comment (vinitshahdeo) [inline src/Components/Impact.js]: @Meghag5 The below component is repeated multiple times, can't we have a map over a JSON?
```js
<ImpactCard
   impact={{
       name: "Open-Source Journey",
       imgUrl: "image3.svg",
       para1:
          "Get your journey in open-source started with our open-source projects!",
                para2: "",
    }}
 />
```

File: src/Components/Impact.js [hunk 2]
```diff
@@ -33,6 +33,50 @@ export default class Impact extends Component {
                 para2: "right way!",
               }}
             />
+          </Row>
+          <Row
+            className="justify-content-md-center"
+            style={{ marginTop: "5px" }}
```
Comment (vinitshahdeo) [inline src/Components/Impact.js]: @Meghag5 Please try to avoid inline styling. It's not a good practice as it will be a problem when we scale our system.

**Review by vinitshahdeo — CHANGES_REQUESTED** [review]
@Meghag5 Hey, I've added a few comments, please do have a look!

File: src/Components/Impact.js [hunk 3]
```diff
@@ -4,7 +4,61 @@ import ImpactCard from "./ImpactCard";
 import styles from "../Css/Impact.module.css";
 
 export default class Impact extends Component {
+  constructor() {
+    super();
+    this.state = {
+      people: [
```
Comment (Manvityagi) [inline src/Components/Impact.js]: @Meghag5 Give meaningful names to vars always, change people to impactCards

**Review by Manvityagi — CHANGES_REQUESTED** [review]

File: src/Components/Impact.js [hunk 4]
```diff
@@ -4,7 +4,61 @@ import ImpactCard from "./ImpactCard";
 import styles from "../Css/Impact.module.css";
 
 export default class Impact extends Component {
+  constructor() {
+    super();
+    this.state = {
+      impactcards: [
```
Comment (Manvityagi) [inline src/Components/Impact.js]: impactCards not impactcards 
That's the camelCase convention 

@Meghag5

**Review by Manvityagi — CHANGES_REQUESTED** [review]

**Pull request merged**